### ISSUE TITLE
`generated.go` references an incorrect import to `model`

### ISSUE THREAD

**Issue by gbdubs** [Silicon-Ally/rules_gqlgen#5]
The following configuration (from silicon starter) yeilds the following generated code, which fails to compile. I suspect it's an easy fix, but I don't know where to begin looking.

in `silicon-starter/cmd/server/BUILD.bazel`:
```
gqlgen(
    name = "gql",
    base_importpath = "github.com/Silicon-Ally/silicon-starter/cmd/server/graph",
    schemas = ["//cmd/server/graph:schema.graphqls"],
    visibility = ["//visibility:public"],
)
```

Yields `silicon-starter/bazel-bin/cmd/server/graph/generated/generated.go`
```
import "github.com/Silicon-Ally/rules_gqlgen/cmd/server/model"
```

The compiler (rightly?) complains:

```
compilepkg: missing strict dependencies:
        /home/gbw/.cache/bazel/_bazel_gbw/9275d41ea74178e1a6f7154e40e03e56/sandbox/linux-sandbox/632/execroot/__main__/bazel-out/k8-fastbuild/bin/cmd/server/graph/generated/generated.go: import of "github.com/Silicon-Ally/rules_gqlgen/cmd/server/model"
        ```

**Comment by bcspragu** [comment]
Hmm, we _do_ specify the model as a dependency though:

https://github.com/Silicon-Ally/rules_gqlgen/blob/b58e9db42c56f0a259f0ee7f3f423961e39e562f/gqlgen/def.bzl#L97

I'll keep digging

**Comment by gbdubs** [comment]
I honestly don't understand enough about this bug to know whether or not
You're already beyond my comprehension here, but I think the error is that
we're referencing the wrong package name, rather than having silicon
starter in the package name it has g equal gem

**Comment by bcspragu** [comment]
Ah, figured it out, it was a path issue. Not sure how the example in `rules_gqlgen` worked, but I'll send you a PR there, cut a new release, and pull that into here

**Comment by bcspragu** [comment]
Closed with #6
